DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Administrator/DeactivateAdmin.cshtml

```diff
@@ -2,16 +2,20 @@
 @model DeactivateAdminViewModel
 
 @{
-  ViewData["Title"] = !ViewData.ModelState.IsValid ? "Error: " : "" + "Deactivate Admin Account";
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = errorHasOccurred ? "Error: " : "" + "Deactivate Admin Account";
 }
 
 <div class="nhsuk-grid-row">
   <div class="nhsuk-grid-column-full">
+    @if (errorHasOccurred) {
+      <vc:error-summary order-of-property-names="@(new[] { nameof(Model.Confirm) })" />
+    }
     <h1 class="nhsuk-heading-l">Are you sure you would like to deactivate this admin account?</h1>
   </div>
 </div>
 
-<div class="nhsuk-grid-row nhsuk-u-margin-6">
+<div class="nhsuk-grid-row">

```

Comment:
We still need some vertical spacing applied here as per comment 5 of my large comment. Can we have some screenshots once you've done that.

Reply:
I have added vertical spacing. Hope this look ok.

Desktop version 
![image](https://user-images.githubusercontent.com/90783892/142587593-50ba6d04-b409-470c-bf31-e76414ef3f42.png)

Mobile versioin
![image](https://user-images.githubusercontent.com/90783892/142587674-505cc4f4-8a8b-41d9-913d-adbb2e624fea.png)

Reply:
Not sure it's made much difference, but I guess it's fine